man/systemd.special.xml

```diff
@@ -667,6 +671,17 @@
           default).</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><filename>wakeup.target</filename></term>
+        <listitem>
+          <para>A special target unit that is started after
+          <filename>suspend.target</filename>,
+          <filename>hibernate.target</filename> and
+          <filename>hybrid-sleep.target</filename>.  This can be used
+          to pull in units and order them after the system wakes up
+          from sleep.</para>
+        </listitem>
```

Comment:
@keszybz 
> tell users to specify also `After=hibernate.target sleep.target hybrid-sleep.target`
If this is enough, why do we need special `wakeup.target` at all? `WantedBy=sleep.target` with `After=sleep.target`. What adding additional target allows to do that this does not?